openapi/sms/v1-review/partner-service.yaml

```diff
@@ -0,0 +1,364 @@
+openapi: 3.0.3
+info:
+  title: SAP Emarsys SMS Partner Service
+  description: |
+    An SAP Emarsys SMS Partner Service allows partners to host SMS message sends that are triggered
+    by an SAP Emarsys SMS campaign launch. This OpenAPI specification describes the required Partner service endpoints.
+  version: 0.0.1
+  contact:
+    name: SAP Emarsys SMS Team
+    email: team-sms@emarsys.com
+x-sap-shortText: Configure, test client setup and sends SMS messages.
+servers:
+  - url: https://api.sms-partner.io/sms/partner/v1
+paths:
+  /clients/{clientId}/configuration:
+    summary: Manage a client configuration within a Partner service
+    parameters:
+      - $ref: './common.yaml#/components/parameters/clientIdPathParam'
+    put:
```

Comment:
Isn't this a PATCH? A PUT is just edit and not create.

Reply:
I agree, but unfortunately this is already being used and the decision to merge these endpoints wasn't mine. Let's see if we can ask CustDev if they are okay with us changing it